src/mixins/labels.js

```diff
@@ -16,10 +16,31 @@ export default {
       }
 
       if (c.type === 'group') {
-        return this.$store.getters['channels/otherMembersOf'](c.channelID, this.$auth.user.userID)
+        let groupMembers = this.$store.getters['channels/otherMembersOf'](c.channelID, this.$auth.user.userID)
           // Make sure user exists
           .filter(o => this.$store.getters['users/findByID'](o) !== undefined)
-          .map(this.labelUser).join(', ')
+
+        if (memberStatus) {
+          // Member names show status (grey for offline, black for online)
+          return groupMembers.map(o => {
+            const label = this.labelUser(o)
+            if (this.$store.getters['users/isPresent'](o)) {
+              return label
+            } else {
+              return `<span class="member-offline">${label}</span>`
```

Comment:
Let's not solve this problem with a hammer :)
While extracting this into a new component, move this into component's template.